compiler/circle-execution-plan/src/ExecutionPlanner.cpp

```diff
@@ -86,18 +86,195 @@ void create_allocation_node(Json::Value &allocations_node,
   allocations_node.append(allocation_node);
 }
 
+// TODO: Introduce inplace optimization
+bool can_be_inplace_optimization_node(luci::CircleNode *node)
+{
+  switch (node->opcode())
+  {
+    case luci::CircleOpcode::LOGISTIC:
+    case luci::CircleOpcode::RESHAPE:
+    case luci::CircleOpcode::EXPAND_DIMS:
+      return true;
+    default:
+      return false;
+  }
+}
+
 } // namespace
 
-void ExecutionPlanner::make_execution_plan()
+void ExecutionPlanner::make_execution_plan_luci_interpreter()
 {
   get_default_execution_order_plan();
   _required_size = get_offsets_with_greedy_by_size();
+
   for (uint32_t i = 0; i < _ordered_nodes.size(); i++)
   {
     luci::CircleNodeExecutionPlan execution_plan(i, _offsets[i]);
     luci::add_execution_plan(loco::must_cast<luci::CircleNode *>(_ordered_nodes[i]),
                              execution_plan);
   }
+
+  printf("Buffer required memory = %d\n", _required_size);
```

Comment:
Unintended print? or please use logging function not printf

Reply:
done